packages/@tailwindcss-upgrade/src/template/codemods/important.ts

```diff
@@ -3,6 +3,18 @@ import { parseCandidate } from '../../../../tailwindcss/src/candidate'
 import type { DesignSystem } from '../../../../tailwindcss/src/design-system'
 import { printCandidate } from '../candidates'
 
+const QUOTES = ['"', "'", '`']
+const LOGICAL_OPERATORS = ['&&', '||', '===', '==', '!=', '!==', '>', '>=', '<', '<=']
+const CONDITIONAL_TEMPLATE_SYNTAX = [
+  // Vue
+  /v-if=['"]$/,
+  /v-show=['"]$/,
+
+  // Alpine
+  /x-show=['"]$/,
+  /x-if=['"]$/,
```

Comment:
Pedantic af but should we put these in the same order as above?

Reply:
Sorted alphabetically now